src/main/kotlin/me/partlysanestudios/partlysaneskies/data/location/Point2d.kt

```diff
@@ -3,7 +3,7 @@
 // See LICENSE for copyright and license notices.
 //
 
-package me.partlysanestudios.partlysaneskies.features.farming.endoffarmnotifer.points
+package me.partlysanestudios.partlysaneskies.data.location
```

Comment:
Fyi this should be moved to render when that gets merged

Reply:
right